media/media_library/media_box.go

```diff
@@ -77,15 +79,25 @@ func (mediaBox *MediaBox) IsSVG() bool {
 }
 
 func (mediaBox *MediaBox) URL(styles ...string) (s string) {
```

Comment:
This is a syntactic sugar. You can pass no parameter to get the default URL of an image. Or pass 1 style parameter to fetch URL of certain style image.  If you pass multiple parameters, only the first one will be used.